app/services/generate_training_data.rb

```diff
@@ -0,0 +1,48 @@
+require "openai"
+
+class GenerateTrainingData
+  def initialize(import_id)
+    @client = OpenAI::Client.new
+    @import_id = import_id
+  end
+
+  def call
+    response = @client.chat(
+      parameters: {
+        model: "gpt-4",
```

Comment:
Do we need to use gpt-4 or would 3.5 suffice?

Reply:
3.5 could probably be okay. I didn't test it out yet